guides/oidc/oidc.md

```diff
@@ -0,0 +1,116 @@
+# OpenID Connect
+
+The OpenID Connect (OIDC) feature allows third-party applications to verify the identity of the end-user and to obtain basic user profile information. Instead of managing usernames and passwords directly **trento-web console will rely on an identity provider (IDP) to authenticate users**.
+
+## Enabling OIDC
+
+The OIDC feature is **disabled by default**.
+
+Provide the following environment variable to enable OIDC feature when starting Trento.
+
+```
+ENABLE_OIDC=true
+OIDC_CLIENT_ID=<<OIDC_CLIENT_ID>>
+OIDC_CLIENT_SECRET=<<OIDC_CLIENT_SECRET>>
+OIDC_BASE_URL=<<OIDC_BASE_URL>>
+OIDC_CALLBACK_URL=<<OIDC_CALLBACK_URL>>
+```
+
+## Enabling OIDC in Development
+
+Enable OpenID Connect (OIDC) in the development environment using Docker and [Keycloak](https://github.com/keycloak/keycloak) as a simulated Identity Provider (IDP).
+
+### Enable OIDC in dev environment by editing `config/dev.exs`
```

Comment:
You should never edit the `dev.exs` file.
You should create your own `dev.local.exs` file and add there the needed changes. that will overwrite the original dev configuration.

If you simply want to enable and use our docker compose keycloak , the file content would be:
```
import Config

config :trento, :oidc, enabled: true
```

Reply:
Very fair point :+1:  , i made my life harder here xD